src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -258,7 +258,16 @@ private static CommandLineProgram extractCommandLineProgram(final String[] args,
                 if (null == property) {
                     if (missingAnnotationClasses.isEmpty()) missingAnnotationClasses += clazz.getSimpleName();
                     else missingAnnotationClasses += ", " + clazz.getSimpleName();
-                } else if (!property.omitFromCommandLine()) { /** We should check for missing annotations later **/
+                } else if (property.omitFromCommandLine()) {
+                    // for classes that should be removed from the command line, run directly if is it called in the command line
+                    if (args.length > 1 && clazz.getSimpleName().equals(args[0])) {
+                        try {
+                            return (CommandLineProgram) clazz.newInstance();
+                        } catch (final InstantiationException | IllegalAccessException e) {
+                            throw new RuntimeException(e);
+                        }
+                    }
```

Comment:
Rather than short-circuiting all of the logic below (name collision tests, -h, etc.), and duplicating this code, it would be better to just restrict the specific behavior we want changed, which I think is only to not include the tool in printUsage output. I think it would be better to just have printUsage filter them out.